frontend/javascripts/oxalis/model/reducers/volumetracing_reducer.js

```diff
@@ -132,41 +133,49 @@ function handleUpdateSegment(state: OxalisState, action: UpdateSegmentAction) {
   });
 }
 
+export function serverVolumeToClientVolumeTracing(tracing: ServerVolumeTracing): VolumeTracing {
+  const maxCellId = tracing.largestSegmentId;
+  const userBoundingBoxes = convertUserBoundingBoxesFromServerToFrontend(tracing.userBoundingBoxes);
+  const volumeTracing = {
+    createdTimestamp: tracing.createdTimestamp,
+    type: "volume",
+    segments: new DiffableMap(
+      tracing.segments.map(segment => [
+        segment.segmentId,
+        {
+          ...segment,
+          id: segment.segmentId,
+          somePosition: Utils.point3ToVector3(segment.anchorPosition),
+        },
+      ]),
+    ),
+    activeCellId: 0,
+    lastCentroid: null,
+    contourTracingMode: ContourModeEnum.DRAW,
+    contourList: [],
+    maxCellId,
+    tracingId: tracing.id,
+    version: tracing.version,
+    boundingBox: convertServerBoundingBoxToFrontend(tracing.boundingBox),
+    fallbackLayer: tracing.fallbackLayer,
+    userBoundingBoxes,
+  };
+  return volumeTracing;
+}
+
 function VolumeTracingReducer(state: OxalisState, action: VolumeTracingAction): OxalisState {
   switch (action.type) {
     case "INITIALIZE_VOLUMETRACING": {
       // As the frontend doesn't know all cells, we have to keep track of the highest id
       // and cannot compute it
```

Comment:
This should be moved to the `serverVolumeToClientVolumeTracing` function as well